docker/Makefile

```diff
@@ -46,7 +46,11 @@ deployable-app: build-ui
 
 .PHONY: prod
 prod: build-ui
-	docker build -f ui-builder/Dockerfile.prod -t lf-ui-builder:prod .
+	docker build -f ui-builder/Dockerfile.prod . -t lf-ui-builder:prod
+	docker build app-base -t lf-app-base
+	docker build -f app-base/Dockerfile.prod app-base -t lf-app-base:prod
+	docker-compose build app
```

Comment:
It looks like you are taking advantage of `docker-compose build` so you don't have to write `docker build` lines, correct?  It's fine the way it is presumably, but I note that it's opaque at this point what docker-compose is actually building and which images you will actually use.

Reply:
it has to do with whether or not the existing definition in the `docker-compose.yml` will work as-is or not.  For example, line 49 requires a different `Dockerfile` than the one already established in the `docker-compose.yml` so it seems silly to add another entry just to reference a different `Dockerfile`.